BooruSharp.Others/Pixiv.cs

```diff
@@ -14,16 +14,39 @@
 
 namespace BooruSharp.Others
 {
+    /// <summary>
+    /// Pixiv.
+    /// <para>https://www.pixiv.net/</para>
+    /// </summary>
     public class Pixiv : ABooru
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Pixiv"/> class.
+        /// </summary>
         public Pixiv() : base("app-api.pixiv.net", (UrlFormat)(-1), BooruOptions.noComment | BooruOptions.noLastComments | BooruOptions.noMultipleRandom |
                 BooruOptions.noPostByMd5 | BooruOptions.noRelated | BooruOptions.noTagById | BooruOptions.noWiki | BooruOptions.noEmptyPostSearch)
         {
             AccessToken = null;
         }
 
+        /// <summary>
+        /// Sends a login API request using specified
+        /// <paramref name="username"/> and <paramref name="password"/>.
+        /// </summary>
+        /// <param name="username">Pixiv user name.</param>
+        /// <param name="password">Pixiv user password.</param>
+        /// <returns>The task object representing the asynchronous operation.</returns>
```

Comment:
Is it really useful to explain what a Task is?

Reply:
Microsoft does this everywhere `Task` is returned, so I did the same.